debian/control

```diff
@@ -4,18 +4,23 @@ Priority: extra
 Maintainer: Sipwise Development Team <support@sipwise.com>
 Build-Depends: debhelper (>= 5),
                iptables-dev (>= 1.4),
-               libavcodec-dev,
-               libavfilter-dev,
-               libavformat-dev,
-               libavresample-dev,
-               libavutil-dev,
+               libavcodec-dev (>= 6:10),
+               libavcodec55 (>= 6:10) | libavcodec-dev (>= 6:11),
+               libavfilter-dev (>= 6:10),
+               libavfilter4 (>= 6:10) | libavfilter-dev (>=6:11),
+               libavformat-dev (>= 6:10),
+               libavformat55 (>= 6:10) | libavformat-dev (>=6:11),
```

Comment:
These dependencies in the style: "libavformat-dev (>= 6:10), libavformat55 (>= 6:10) | libavformat-dev (>=6:11)," are still not correct, because you are assuming private linking details of ffmpeg or libav. You might end up with a libavcodec-dev package that provides a libavcodec.so file that points to something like libavcodec.so.56, even if you have requested here libavcodec55 as Build-Depends.

If the code does not build with older packages, or you need features only available in later versions then adding a version restriction seems entirely legit, but that would be simply adding "(>= 6:10)" or similar. But again it would require verifying if this does not build with older versions. But the hardcoding of the libfooNN is just wrong here.